src/pages/researcher.pug

```diff
@@ -0,0 +1,90 @@
+
+      figure
+        img(src="/assets/images/researcher/inline-2.webp" alt="Chris and Mike" onerror="this.onerror=null; this.src='/assets/images/researcher/inline-2.jpg'")
+        figcaption Chris and Mike on a face-to-face team catch-up day
+
+      .grid.grid--two-thirds
+        .grid__item
+          h4 Primary responsibilities
+          ul.styled-list.styled-list--inside-lg
+            li.styled-list__item Devising and undertaking research to support Leaf's marketing strategy.
+            li.styled-list__item Setting-up and running qualitative and quantitative user research activities.
+            li.styled-list__item Communicating your findings effectively to the team and external content editors.
+            li.styled-list__item Conducting interviews and creating relationships with key speakholders.
+            li.styled-list__item Sourcing individuals from appropriate demographics for user testing sessions.
+            br
+            p <strong>Note: </strong> This is a <em>startupesque</em> role, where you'll be asked to wear many hats, work outside your comfort zome, and have complete ownership of the area.  
+          
+          h4 About you
+          ul.styled-list.styled-list--inside-lg
+            li.styled-list__item You have background in research, with at least 3 years industry experience.
+            li.styled-list__item You thrive being a big fish in a smaller pond (you'll be the 6<sup>th</sup> member of the team).
+            li.styled-list__item You're a natural communicator; being able to articulate your point, both internally and to users, is essential.
+            li.styled-list__item You have a portfolio of case studies that show-off your best moves.
+            li.styled-list__item You love to teach what you know, and learn what you don't.
+            li.styled-list__item You're bursting with passion, and have a natural drive to better yourself.
+            
+          h5 Bonus points if…
+          ul.styled-list.styled-list--inside-lg
+            li.styled-list__item You have experience conducting research remotely.
+            li.styled-list__item You're able to lead, rather than be lead.
+            li.styled-list__item You have real-World experience or are further educated in psychology or a related field.        
+      
+          h4 Location
+          p This is a remote position, although a few times a year we like everyone to drop into Canterbury for a bit of face-time with the rest of the team. The company themselves are based in Canterbury; famous for its cathedrals, cobbles and tales.
+
+          h4 How to apply
+          p If this role felt like a bit of you, we want to know about it. Tell us a little about yourself, along with a CV, some examples of your work, and let us know why you'd be a good fit.
```

Comment:
"If this role felt like a bit of you" reads a bit strangely to me. I think it should be present-tense, and I'm not really familiar with the expression "feels like a bit of you". Perhaps it could be swapped for something easier to understand?